Doc/library/pdb.rst

```diff
@@ -252,6 +252,10 @@ change a variable or call a function.  When an exception occurs in such a
 statement, the exception name is printed but the debugger's state is not
 changed.
 
+.. versionchanged:: 3.13
+   Expressions/Statements whose prefix is a pdb command are now correctly
+   identified and executed.
+
```

Comment:
This should probably be at the bottom, just before the entry for 3.12.

Reply:
The entry for 3.12 is the one for convenience variable and that's specifically for the content above. I put the version changed at the current position because I think it's describing the paragraph above:

> Commands that the debugger doesn’t recognize are assumed to be Python statements and are executed in the context of the program being debugged

Do you still want me to move it to a better place? The following paragraphs are about
* Aliases
* Multiple commands
* Convenience variables
* pdbrc

Reply:
Yes, you're right, it belongs here.